third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2072,12 +2073,12 @@ namespace posix {
 
 #if GTEST_OS_STARBOARD
 
-using StatStruct = SbFileInfo;
+typedef struct stat StatStruct;
```

Comment:
Please leave it as `using StatStruct = stat`;

Reply:
https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h;l=2210

We also use `typedef struct stat StatStruct` in a couple of other places in the gtest file -- is there a benefit to using `using` instead?

Reply:
No other benefit than consistent style both in template code and outside of it. If the compiler barks, it's fine to leave as is.